sdk/storage/Azure.Storage.Files.Shares/api/Azure.Storage.Files.Shares.netstandard2.0.cs

```diff
@@ -611,6 +615,20 @@ public partial class ShareFileCopyInfo
         public Azure.ETag ETag { get { throw null; } }
         public System.DateTimeOffset LastModified { get { throw null; } }
     }
+    public partial class ShareFileCopyOptions
+    {
+        public ShareFileCopyOptions() { }
+        public Azure.Storage.Files.Shares.Models.ShareFileRequestConditions Conditions { get { throw null; } set { } }
+        public bool? CopySourceFileAttributes { get { throw null; } set { } }
```

Comment:
We probably shouldn't make these nullable if we stick with bools.  That's very useful to say "rely on the service default", but this is a client only shortcut to workaround C# types that can't be both DateTimeOffset and String.